apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -141,7 +143,18 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     process.exitCode = 0;
   }
 
-  protected initializePlugins(pluginSpecifiers: ReadonlyArray<string>): void {
+  private _normalizeCwd(): void {
+    const buildFolder: string = this._heftConfiguration.buildFolder;
+    this.terminal.writeVerboseLine(`Project build folder is "${buildFolder}"`);
+    const currentCwd: string = process.cwd();
+    if (currentCwd !== buildFolder) {
+      // Update the CWD to the project's build root. Some tools, like Jest, use process.cwd()
+      this.terminal.writeVerboseLine(`CWD is "${currentCwd}". Normalizing to project build folder.`);
```

Comment:
This seems standard/expected enough that maybe we don't need to call it out explicitly in the logging?

Instead just say something like `Project folder is: ${buildFolder}`, and maybe as a non-verbose log item.

Reply:
I like calling out that we're explicitly changing/correcting the CWD. It seems like it could be useful for troubleshooting.

I'll change the other to non-verbose.